common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -1304,22 +1305,22 @@
     "figura.docs.text_task.set_alignment": "Sets this Text alignment\nCan be either \"LEFT\", \"RIGHT\" or \"CENTER\"\nDefault \"LEFT\"",
     "figura.docs.text_task.has_shadow": "Checks if this task text has shadow",
     "figura.docs.text_task.set_shadow": "Toggles if the Text should render with a drop shadow\nNot compatible with \"Outline\" mode",
-    "figura.docs.text_task.has_outline": "Checks if this task text has outline",
-    "figura.docs.text_task.set_outline": "Toggles if the Text should render with a outline\nAlways renders at full bright\nNot compatible with \"Shadow\" and \"Emissive\" modes",
+    "figura.docs.text_task.has_outline": "Checks if this task text has an outline",
+    "figura.docs.text_task.set_outline": "Toggles, if the text should render with an outline\nAlways, renders at full bright\nNot compatible with \"Shadow\" and \"Emissive\" modes",
     "figura.docs.text_task.get_outline_color": "Gets this tasks text outline color",
     "figura.docs.text_task.set_outline_color": "Sets the outline color this Text should render\nOnly compatible with \"Outline\" mode",
     "figura.docs.text_task.get_width": "Gets this Text max width\nDefault 0",
-    "figura.docs.text_task.set_width": "Sets this Text max width, wrapping the text into multiple lines\nWidth of 0 or less do not wraps the text\nDefault 0",
+    "figura.docs.text_task.set_width": "Sets this Text max width, wrapping the text into multiple lines\nWidth of 0 or less do not wrap the text\nDefault 0",
     "figura.docs.text_task.has_wrap": "Check if this Text should wrap lines",
     "figura.docs.text_task.set_wrap": "Sets if this Text should wrap lines",
     "figura.docs.text_task.is_see_through": "Check if this Text can be seen behind walls\nDefault false",
     "figura.docs.text_task.set_see_through": "Sets if this Text can be seen behind walls\nDefault false",
     "figura.docs.text_task.has_background": "Check if this Text should render its background\nDefault false",
     "figura.docs.text_task.set_background": "Sets if this Text should render its background\nDefault false",
```

Similar changes made as above.
```suggestion
    "figura.docs.text_task.has_wrap": "Check if this text should wrap lines",
    "figura.docs.text_task.set_wrap": "Sets if this text should wrap lines",
    "figura.docs.text_task.is_see_through": "Check if this text can be seen behind walls\nDefaults to false",
    "figura.docs.text_task.set_see_through": "Sets if this text can be seen behind walls\nDefaults to false",
    "figura.docs.text_task.has_background": "Check if this text should render its background\nDefaults to false",
    "figura.docs.text_task.set_background": "Sets if this text should render its background\nDefaults to false",
```